app/main/menu/menus.js

```diff
@@ -21,6 +21,7 @@ import electron from 'electron';
 import items from './submenu-items';
 
 const APP_NAME = electron.app.getName();
+const isProduction = process.env.NODE_ENV === 'production';
```

Comment:
This is the second time I'm seeing us checking NODE_ENV instead of the build config for determining if something is in production. Unless I'm missing something, we should use the build config instead. This environment variable doesn't guarantee a packaged build.

There are good reasons for using a development flag in the build config, we can reiterate outside of this PR if needed.

A much better alternative would be to have a 'production' property on the build config. This would also fix the annoying default browser prompt in local production builds.